packages/hardhat-core/src/types/builtin-tasks/node.ts

```diff
@@ -0,0 +1,10 @@
+import { EIP1193Provider } from "../provider";
+
+export interface JsonRpcServer {
+  getProvider(name: string): EIP1193Provider;
```

Comment:
This is only for testing purposes, so I don't think it should be placed here.

Reply:
Fixed :+1: `close` isn't used in the task either, but I guess it makes sense to have it in the interface anyway, wdyt?

Reply:
Yes, I think that can be really useful for people overriding the tasks.